sdk/go/common/util/cmdutil/console.go

```diff
@@ -256,3 +292,83 @@ func max(a, b int) int {
 	}
 	return b
 }
+
+// readConsoleModel drives a bubbletea widget that reads from the console.
+type readConsoleModel struct {
+	input  textinput.Model
+	secret bool
+
+	// Canceled is set to true when the model finishes
+	// if the user canceled the operation by pressing Ctrl-C or Esc.
+	Canceled bool
+
+	// Value is the user's response to the prompt.
+	Value string
+}
+
+var _ tea.Model = readConsoleModel{}
+
+func newReadConsoleModel(prompt string, secret bool) readConsoleModel {
+	input := textinput.New()
+	input.Cursor.Style = lipgloss.NewStyle().Foreground(lipgloss.Color("205"))
```

Comment:
Whats 205?

Reply:
It's the color of the cursor I chose above.

<img width="129" alt="image" src="https://github.com/pulumi/pulumi/assets/41730/7d9875b5-9a56-4529-ad91-2b4d21c9596b">

We can change it to anything, but something loud and obvious made sense at the time.

Reply:
`// 205 = hot pink cursor`

Reply:
Derp. Yeah, makes sense to add a comment.  Done.